supply.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+from typing import Any, Dict
+from boto3 import client
+
+from constants import base_payload, decimal_places
+
+lambda_client = client('lambda', region_name='us-east-1')
+
+
+def get_status_payload() -> Dict[str, Any]:
+    """ Get payload data from status lambda function
+    """
+    invoke_response = lambda_client.invoke(FunctionName='status-economic')
```

Comment:
Do we have to call the other lambda? Can't we just import the other file and call the function directly?

Reply:
In my first solution (when I had the cache enabled) I was executing the request for the API gateway, so I could get the updated value from the cache.

When we removed the cache, I decided to invoke the status lambda, to decrease the lambda package (because I wouldn't need to use the `requests` library) - see image of the code size.

![image](https://user-images.githubusercontent.com/3298774/80154371-cc27c080-8595-11ea-9c96-de513cf2c36a.png)

But I guess it doesn't make much difference because I still need to call the other lambda, that has the big code size.

So I think I have two possible solutions:

1. Go back to the API request, so the code is already ready if/when we enable the cache.
2. Refactor the lambda code and call the python function for the status directly.

What do you think?

Reply:
I'm not sure. Does internal call count as a regular call? I think we can start as it is, but we may need to fix it later if it doubles the call counter.

Reply:
It does count as double. If we decide to use cache in the future we won't escape this double count (in the API Gateway) but right now I guess we could avoid it.

So I will create an util file with the method.

Reply:
Done. Can you see if it's good?

Reply:
It is. Thanks!